DependencyInjection/Configuration.php

```diff
@@ -25,12 +25,23 @@ public function getConfigTreeBuilder()
             ->addDefaultsIfNotSet()
             ->children()
                 ->scalarNode('private_key_path')
+                    ->setDeprecated('The "%path%.%node%" configuration key is deprecated since version 2.5. Use "%path%.secret_key%" instead.')
```

Comment:
I think the new configuration key should contain 2 or 4 `%` symbols, not 3, right?

Reply:
good catch